dee-pabst/week5/gift-network-app/app/models/item.rb

```diff
@@ -0,0 +1,5 @@
+class Item < ApplicationRecord
+
+    belongs_to :member
```

Comment:
You need to add `optional: true` as this expects a `member_id` to be attached at creation (also this is missing from your form, unless you want it to be writing in your controller using - `current.user` (not sure if we were taught this at this point. 

so for now, adding optional true will create items without member ids